handbook.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: contribute
+title: Handbook
+---
+
+# How we make things
+
+## Branching and SCM
+
+We use git for source control. We follow this model in what we call the core projects ([Foreman](https://github.com/theforeman/foreman), [Smart Proxy](https://github.com/theforeman/smart-proxy), [Foreman installer](https://github.com/theforeman/foreman-installer) and [Foreman SELinux](https://github.com/foreman-selinux)) for branching:
+
+* **develop**
+    * New pull requests should be made against this branch.
+    * All pull requests are merged here.
+    * Contains nightlies code. It's relatively stable, but we make no promises about it.
+    * [`test_develop` in Jenkins](http://ci.theforeman.org/job/test_develop/) ensures our tests are passing at all times for various configurations.
+* **release-stable (i.e. 1.10-stable)** -
+    * Individual developers do not need to maintain this, it's part of our [release process](http://projects.theforeman.org/projects/foreman/wiki/Release_Process#Preparing-a-release)
+    * Code is cherry-picked using `git cherry-pick -x` into this branch.
+    * It contains the latest stable code for a release cycle, 1.8-stable contains 1.8.1, 1.8.2, 1.8.3...
+    * Typically, we will announce the start of a new branch
+
+And we tag releases:
+
+* **release (i.e. 1.10)**
+    * Code is cherry-picked using `git cherry-pick -x` into this tag.
+    * It contains the latest stable code for a *specific release*, for instance, 1.10.0.
```

Comment:
This three line section is a bit confusing combined with what was above it -- maybe just add a line to the above about how we tag each stable release point.